### ISSUE TITLE
Add cluster analytics/stats endpoint to the API

### ISSUE THREAD

**Issue by hexedpackets** [spotify/heroic#714]
Right now its hard to discover information about how a Heroic cluster is being used, requiring something like a separate QueryLog analytics setup. Some of this information is actually pretty discoverable within Heroic itself, and just needs a decent way of querying it.

We already have [an API endpoint that can return the total number of active timeseries](https://spotify.github.io/heroic/docs/api/post-metadata-series-count). Some other possibilities that have varying levels of difficulty:

- number of active requests by endpoint
- requests that have been running for over a set amount of time (similar to something like [pg_stat_statements](https://www.postgresql.org/docs/current/pgstatstatements.html))
- echo back current config
- information on the various pipeline connections (PubSub, Cassandra, BigTable)

As an example for comparison, OpenTSDB has an endpoint that returns [all active queries and the 256 most recently completed ones](http://opentsdb.net/docs/build/html/api_http/stats/query.html).

**Comment by sming** [comment]
Hi William, is this part of Heroic Stability work?

**Comment by hexedpackets** [comment]
@sming Not directly, although having this kind of data would help us make more informed decisions when prioritizing our efforts.

**Comment by sming** [comment]
Yes, it looked like a "supporting cast" kind of ticket. Will message you on Slack.
